ros_network_viz/ros_network_viz.py

```diff
@@ -82,26 +82,42 @@ def __init__(self, qos_profile):
             ])
             self.setToolTip(text)
 
+        # Pens used to draw the connection line in selected and unselected states
         self._pen = QtGui.QPen(convert_hex_to_color(COLOR_PALETTE['connection_line']), 3)
-
         self._pen_sel = QtGui.QPen(
             convert_hex_to_color(COLOR_PALETTE['connection_line_highlight']), 5)
 
+        # Brushes used to fill the arrow in selected and unselected states
+        self._brush = QtGui.QBrush(convert_hex_to_color(COLOR_PALETTE['connection_line']))
+        self._brush_sel = QtGui.QBrush(convert_hex_to_color(COLOR_PALETTE['connection_line_highlight']))
+
         self.setPen(self._pen)
 
+        # The connection line begins in the unselected state
+        self._selected = False
+        self._current_brush = self._brush
+
+        # The endpoints of the connection line aren't known until update_path is called
+        self._arrow_location = None
+
     # PyQt method override
     def hoverEnterEvent(self, event):
         self.setPen(self._pen_sel)
+        self._current_brush = self._brush_sel
+        self._selected = True
```

Comment:
Do we actually need this `self._selected` variable?  I see it being set, but there are no comparisons that use it.

Reply:
No, left around from a previous version of the code I was working on. Removed.